Dockerfile.test-infra

```diff
@@ -4,6 +4,8 @@ FROM quay.io/app-sre/centos:8
 
 RUN yum -y install make python3 libvirt-client libvirt-devel libguestfs-tools gcc unzip wget curl git python3-devel podman httpd-tools && yum clean all;
 
+RUN wget https://rpmfind.net/linux/centos/8.3.2011/BaseOS/x86_64/os/Packages/libgcrypt-1.8.5-4.el8.x86_64.rpm &&  yum -y install libgcrypt-1.8.5-4.el8.x86_64.rpm && rm -f libgcrypt-1.8.5-3.el8.x86_64.rpm && yum clean all;
```

Comment:
typo in `rm -f libgcrypt-1.8.5-3.el8.x86_64.rpm` should be `rm -f libgcrypt-1.8.5-4.el8.x86_64.rpm`